meetings/2024-07/july-29.md

```diff
@@ -0,0 +1,1054 @@
+
+JKP: I will cover is testing repository which is supported by Igalia, and so we have the entire specification done in unit test and there is additional areas that need to be flushed out but it is near completion, and it has already been integrated into Mozilla search library and that is dev tools, and I think that is being integrated into Chrome. So now our work has continued to add test coverage for the upcoming Scopes proposal. Cool and last thing kind of a heads up on these that we are planning on coming to the October plenary with the clean up specification and hop hoping proposal for TG4 and we will make sure that is centered around that proposal, and thank you so much.
+
+## TG4 convenors appointment
+
+Presenter: Chris de Almeida (CDA )
+
+- no proposal
+- no slides
+
+Private discussion
+
+CDA: Welcome back convener NRO, you have been elected by acclamation.
+
+## TG5: Experiments in Programming Language Standardization
+
+Presenter: Mikhail Barash (MBH)
+
+- no proposal
+- no slides
+
+MBH: Hello, so just a brief update this time. So, we still have our meetings every month. We will arrange the TG5 Workshop co-located with the plenary in Tokyo, this will be on Friday the 11th of October, the first half of the day. It will be hosted by the computing software group at the University of Tokyo. So we will talk about formalization of the grammar models that are used in the ECMA262 spec. This is a topic relevant both to that research group, and to the TC39 committee. And also, KAIST promised to talk about their current work in Wasm SpecTec. So everyone is welcome to attend the workshop, and the registration link is in the Reflector post with the information about the next plenary in Tokyo. So yes, are there any questions?
+
+RPR: there is no one in the queue. So if there is any questions, and I will say that thank you for advertising this. Last time we had on the reflector and not everyone saw this, and thank you for this and you can all attend if you wish. On the Tokyo trip. All right we are doing well on the agenda. And onward we go to updates from the code of conduct committee, Chris?
+
+## Updates from the CoC Committee
+
+Presenter: Chris de Almeida (CDA)
+
+- no proposal
+- no slides
+
+CDA: The conduct goes well for the most part. We have not had any new reports and nothing notable in our spaces and disasters course or GetHub.
+
+CDA: We have not – we prefer not to be busy on the Code of Conduct committee, as always, I will repeat my pitch for folks who would like to join the Code of Conduct committee and please reach out to one of on the Code of Conduct committee and the main and is that you are available for period of one hour every two weeks. Often we do not have to meet during that hour because we have nothing to talk about but it is important that it is blocked out on your calendar to we can function as a committee when we need to. So thank you.
```

Comment:
```suggestion
CDA: We have not – we prefer not to be busy on the Code of Conduct committee, as always, I will repeat my pitch for folks who would like to join the Code of Conduct committee and please reach out to one of us on the Code of Conduct committee. The main ask is that you are available for a period of one hour every two weeks. Often we do not have to meet during that hour because we have nothing to talk about but it is important that it is blocked out on your calendar so we can function as a committee when we need to. So thank you.
```